README.md

```diff
@@ -0,0 +1,94 @@
+[![License](https://img.shields.io/:license-apache-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html)
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/73/badge)](https://bestpractices.coreinfrastructure.org/projects/73)
+[![Puppet Forge](https://img.shields.io/puppetforge/v/simp/x2go.svg)](https://forge.puppetlabs.com/simp/x2go)
+[![Puppet Forge Downloads](https://img.shields.io/puppetforge/dt/simp/x2go.svg)](https://forge.puppetlabs.com/simp/x2go)
+[![Build Status](https://travis-ci.org/simp/pupmod-simp-x2go.svg)](https://travis-ci.org/simp/pupmod-simp-x2go)
+
+
+#### Table of Contents
+
+* [Description](#description)
+  * [This is a SIMP module](#this-is-a-simp-module)
+* [Setup](#setup)
+  * [What x2go affects](#what-x2go-affects)
+* [Usage](#usage)
+* [Development](#development)
+  * [Acceptance tests](#acceptance-tests)
+
+## Description
+
+This is a module for managing ``x2go`` server and client installations.
+
+### This is a SIMP module
+
+This module is a component of the [System Integrity Management Platform](https://simp-project.com), a
+compliance-management framework built on Puppet.
+
+If you find any issues, they may be submitted to our [bug tracker](https://simp-project.atlassian.net/).
+
+## Setup
+
+### What x2go affects
+
+The ``x2go`` module is quite minimal, like ``x2go`` itself and simply installs
+the required packages and allows you to configure the most common files with
+safe defaults in place.
+
+## Usage
+
+The ``x2go`` client is installed by default. To disable this, set
+``x2go::client`` to ``false`` in Hiera.
+
+To install and configure the ``x2go`` server, set ``x2go::server`` to ``true`` in Hiera.
+
+``x2go`` requires a functioning window manager on the server to be useful and
```

Comment:
Some good docs on that from the project: https://wiki.x2go.org/doku.php/doc:de-compat